packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -199,10 +199,14 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
     if (prevResizingColumn.current !== resizingColumn && resizingColumn != null && resizingColumn === item.key) {
       wasFocusedOnResizeStart.current = document.activeElement === ref.current;
       startResize(item);
-      focusInput();
+      // Delay focusing input until Android Chrome's delayed click after touchend happens: https://bugs.chromium.org/p/chromium/issues/detail?id=1150073
+      let timeout = setTimeout(() => focusInput(), 0);
```

Comment:
Feels a bit scuffed, ideally we'd prevent the delayed click from happening at all like in https://github.com/adobe/react-spectrum/pull/2942/files but that causes other issues as can be seen in that PR. This previously wasn't an issue since focus was contained in the menu until it fully transitioned out, but we no longer do that since we don't know how long a RAC dropdown might take to transition out.